Persistence/Entities/Package.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace shipping_service.Persistence.Entities
+{
+    public class Package
+    {
+        public ulong Id { get; set; }
+        public DateTime Created { get; set; }
+        public DateTime Modified { get; set; }
+        public string Name { get; set; }
+        public string? Description { get; set; }
+        public Sender? Sender { get; set; }
```

Comment:
A package must have a only and only one sender. It cannot be null.

Reply:
I thought about this and in the first version sender was required. Bu then it occured to me that the sender might delete his account while some of his packages are still in transfer. We could disaloow this, but if we allow it, then this relationship has to be optional (null is set if the sender deletes his account while this package has not yet been delivered).

Reply:
I think we don't need to worry that much. If we gonna get too much in detail from the start, we will soon find ourselves buried on the work of implementing credit cards functionality.
Let's say you cannot delete the accounts for now (and probably at the end of the project, because we don't have much time).

Reply:
If we decide to implement account deletion, we can figure this out later, but now I think having a sender a must is much simpler.

Reply:
Another possibilty: sender deletes it's account = he accepts his package is discarded. He must wait for his shipment to be completed do delete his account. Because if there is no sender, why bother?